libs/utils/src/decorators/ensure-array.decorator.ts

```diff
@@ -0,0 +1,6 @@
+import { Transform } from 'class-transformer';
+
+export const EnsureArray = Transform(({ value }) => {
```

Comment:
how is this different from `@IsArray()`?

Reply:
Good question. This is a transformer, so it ensures that it is an array even if it comes in as not an array. GET Query strings will default to a string not an array if you only send one value, this will transform that single value into an array of 1.

Will add some docs to that effect.